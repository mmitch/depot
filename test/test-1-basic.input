# define funds

+FUND fund_A
+FUND fund_B
+FUND fund_C

# transactions per day

@@ 16.12.2020

fund_A	+1,978 = 32,5    ! 1,21
fund_B	+0,316 = 35,72   ! 1,01

@@ 16.01.2021

fund_A	+1,722 = 32,5    ! 1,21

@@ 15.01.2021

fund_A	+1,812 = 32,5    ! 1,21
fund_C +17,212 = 143,87  ! 4,13

#@@ 26.01.2021

fund_A	-1,000 = 45,4    ! 0,8

@@ 16.02.2021

fund_A	+2,001 = 32,5    ! 1,21
fund_B	+0,288 = 35,73   ! 1,02
