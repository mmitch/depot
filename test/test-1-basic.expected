.---------------------------------------------------------------------------------------------------------.
| ETF    | Anteile      | Wert         | Kurs        (seit Start) | Gewinn        (relativ ) | Stand      |
+--------+--------------+--------------+--------------------------+--------------------------+------------+
| fund_A |    7.513 St. |   122.03 EUR |   16.242 EUR  (  -1.15%) |   -12.81 EUR  (  -9.50%) | 16.02.2021 |
| fund_B |    0.604 St. |    74.93 EUR |  124.062 EUR  (  +9.75%) |     1.45 EUR  (  +1.98%) | 16.02.2021 |
| fund_C |   17.212 St. |   143.87 EUR |    8.359 EUR  (  +0.00%) |    -4.13 EUR  (  -2.79%) | 15.01.2021 |
+--------+--------------+--------------+--------------------------+--------------------------+------------+
| Depot  |              |   340.83 EUR |                          |   -15.49 EUR  (  -4.35%) |            |
'--------+--------------+--------------+--------------------------+--------------------------+------------'
